### PR TITLE
update lassie docs

### PR DESCRIPTION
**Pull request by LexLuthr**
Removed all the go based installation and updated the example CID.

### CODE REVIEW

**Comment by LexLuthr** [comment]
@ElPaisano There was no preview generated or any check performed. Can you please check that and review the PR?

**Review by smagdali — APPROVED** [review]
LGTM

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md [hunk 1]
```diff
@@ -17,11 +17,7 @@ aliases:
 
 ## Lassie
 
-Lassie is a simple retrieval client for IPFS and Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command:
-
-```shell
-lassie fetch <CID>
-```
+Lassie is a simple retrieval client for IPFS and Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command - `lassie fetch <CID>`
```
Comment (johnnymatthews) [inline content/en/basics/how-retrieval-works/basic-retrieval/index.md:20]: Revert this back to the code-block version so users can click the big copy button if they want.

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md [hunk 2]
```diff
@@ -90,29 +54,29 @@ This command uses a `|` to chain two commands together. This will work on Linux
 An example of fetching and extracting a single file, identified by its CID:
 
 ```shell
-lassie fetch -o - bafykbzaceatihez66rzmzuvfx5nqqik73hlphem3dvagmixmay3arvqd66ng6 | car extract - > lidar-data.tar
+lassie fetch -o - bafykbzacecjedcvniq5wylq7cqre7la6diaxwsue5ssy3f3rzftwu3ielspru | car extract - > Juantalks.mp4
```
Comment (johnnymatthews) [inline content/en/basics/how-retrieval-works/basic-retrieval/index.md:57]: What's the benefit of using a talk from Juan?

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Not sure most of this PR is necessary.

What's the reason of using a talk by Juan instead of LIDAR data? The LIDAR data aligns more with the use of Filecoin as a scientific research archive.

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md (hunk 1 above)
Comment (LexLuthr) [reply content/en/basics/how-retrieval-works/basic-retrieval/index.md:20]: This is based on the feedback from users. Having that command well before the installation steps keep tripping them up. Most of the newcomers have complained that it is bad form and it should be after the installation instructions. Another thing to note here is that that was not really a command but just a demonstration of how powerful this tool is. 
@smagdali can probably give you more details on the user experience. This PR is based on the feedback his team collected.

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md (hunk 2 above)
Comment (LexLuthr) [reply content/en/basics/how-retrieval-works/basic-retrieval/index.md:57]: @smagdali You are probably better suited to answer this as well. Although, Juantalk is very slow to retrieve and is available via only 1 source over single protocol.

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md (hunk 1 above)
Comment (johnnymatthews) [reply content/en/basics/how-retrieval-works/basic-retrieval/index.md]: Ok, thanks for clarifying. I've removed the command altogether.

**Comment by johnnymatthews** [comment]
I've made a few changes to this PR, mainly:

- Removed that command from line 20.
- Grammar and spelling fixes.
- Formatting fixes.
- Removed the TODO at the bottom. If you have a TODO item, [create an issue for it](https://github.com/filecoin-project/filecoin-docs/issues.

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md (hunk 2 above)
Comment (johnnymatthews) [reply content/en/basics/how-retrieval-works/basic-retrieval/index.md:56]: I don't want this to get left behind the merge window, so I'm gonna revert this back to the LIDAR data. If there's a pressing reason to use a talk, create a new issue for it.

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md [hunk 3]
```diff
@@ -77,57 +41,56 @@ You now have everything you need to retrieve a file with Lassie and extract the
...
+Fetching bafykbzacecjedcvniq5wylq7cqre7la6diaxwsue5ssy3f3rzftwu3ielspru................................................................................................................................................
+Fetched [bafykbzacecjedcvniq5wylq7cqre7la6diaxwsue5ssy3f3rzftwu3ielspru] from [12D3KooWBwUERBhJPtZ7hg5N3q1DesvJ67xx9RLdSaStBz9Y6Ny8]:
         Duration: 42.259908785s
           Blocks: 144
            Bytes: 143 MiB
 extracted 1 file(s)
 ```
 
-The resulting file is a tar archive:
+The resulting file is an MP4 video file:
```
Comment (LexLuthr) [inline content/en/basics/how-retrieval-works/basic-retrieval/index.md:68]:
```suggestion
The resulting file is a tar archive:
```

File: content/en/basics/how-retrieval-works/basic-retrieval/index.md [hunk 4]
```diff
@@ -77,57 +41,56 @@ You now have everything you need to retrieve a file with Lassie and extract the
...
-```shell
+"`shell
 ls -l
 ```
 
-```shell
+"`shell
 total 143M
--rw-rw-r-- 1 user user 143M Feb 16 11:21 lidar-data.tar
+-rw-rw-r-- 1 user user 143M Feb 16 11:21 Juantalks.mp4
```
Comment (LexLuthr) [inline content/en/basics/how-retrieval-works/basic-retrieval/index.md:76]:
```suggestion
-rw-rw-r-- 1 user user 143M Feb 16 11:21 lidar-data.tar
```

**Comment by LexLuthr** [comment]
@johnnymatthews I am good with lidar data. I have reverted the change back in a couple of places. Let's commit those and merge if you are good.

**Comment by johnnymatthews** [comment]
Haha, looks like we were reverting the content at the same time @LexLuthr 😅. 

This looks good to merge now.

**Review by johnnymatthews — APPROVED** [review]

**Comment by LexLuthr** [comment]
@johnnymatthews I am not authorised to merge. Can you please do the merge as well?

**Pull request merged**